test/integration/dualstack/dualstack_test.go

```diff
@@ -35,997 +35,964 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	kubeapiservertesting "k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
-	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/test/integration/framework"
 	"k8s.io/kubernetes/test/utils/ktesting"
 	netutils "k8s.io/utils/net"
 )
 
 // TestCreateServiceSingleStackIPv4 test the Service dualstackness in an IPv4 SingleStack cluster
 func TestCreateServiceSingleStackIPv4(t *testing.T) {
-	for _, enableMultiServiceCIDR := range []bool{false, true} {
-		for _, disableAllocatorDualWrite := range []bool{false, true} {
```

Comment:
DisableAllocatorDualWrite isn't locked yet, don't we still want to exercise that as true / false? In 1.35 when we lock this true, we'll need to emulate 1.34 to exercise the false case

Reply:
done, these tests are expensive and the main problem was covered with the regressions test we added at the end of the release, but it does not harm to get more coverage in this area, that previously was not good enough to detect the regression